README.md

```diff
@@ -1,2 +1,106 @@
 # unified-user-data-format
+
 Specification for the unified user data format across the various "Tube" apps (NewPipe, FreeTube, Invidious, etc..)
+
+## Intro
+
+* This specification is a very early draft, and prone to regular changes
+* Unless otherwise specified, all fields are assumed to be `Strings`
+
+
+## Main file (`main.json`)
+
+#### `"version"`
+
+The [semantic version](https://semver.org/) of the specification.
+
+Conditions for bumping major/minor/patch version: TBD
+Potential suffixes: `-draft`, TBD
+
+#### `"subscriptions"`
+
+An array of [subscription objects](spec/subscription_object).
+
+#### `"playlists"`
+
+An array of Strings.
+
+Each string refers to the name of a playlist file (without the `playlist_` prefix)
+that was included in the export.
+
+#### `"watch_history_present"`
+
+A Boolean.
+
+Indicates whether the special playlist `watch_history` was included in the export.
```

Comment:
Should we really be doing this? Watch history can have additional fields such as the watched till duration, and additional metadata. I would suggest an additional format to store this.

Reply:
This field is only here to indicate if the `watch_history` file is part of the export.
The format of the watch history can still evolve regardless!

Reply:
@FireMasterK if that confuses you, I can reword that part!
Right now, videos in history and regular playlists are the same objects, but that's definitely going to change ^^

Reply:
> @FireMasterK if that confuses you, I can reword that part! Right now, videos in history and regular playlists are the same objects, but that's definitely going to change ^^

That makes more sense! I think they can be reworded once we finalize a specification for watch history too.

But, in such a scenario, can't we remove this field altogether?

Reply:
I was thinking to make some files optional during the export (like the watch history).
This field's purpose is ti tell the importer whether that file is supposed to be present in the export.